curations/maven/mavengoogle/com.google.android.gms/play-services-gcm.yaml

```diff
@@ -0,0 +1,9 @@
+coordinates:
+  name: play-services-gcm
+  namespace: com.google.android.gms
+  provider: mavengoogle
+  type: maven
+revisions:
+  17.0.0:
+    licensed:
+      declared: Apache-2.0
```

Comment:
```suggestion
      declared: OTHER
```

Reply:
Agree with OTHER. Thank you for the update!